silence/silence.go

```diff
@@ -447,33 +501,14 @@ func QTimeRange(start, end time.Time) QueryParam {
 }
 
 // QMatches returns silences that match the given label set.
-func QMatches(set map[string]string) QueryParam {
+func QMatches(set model.LabelSet) QueryParam {
```

Comment:
I see. We probably have to think about that in terms of the entire repo at a later point. Fine with keeping it as it is for now.